Makefile

```diff
@@ -139,6 +139,8 @@ clean:
 	rm -rf /tmp/avocado*
 	find . -name '*.pyc' -delete
 	find $(AVOCADO_OPTIONAL_PLUGINS) -name '*.egg-info' -exec rm -r {} +
+	# Remove this after 36lts is declared EOL
+	rm -rf avocado.egg-info
```

Comment:
Well how about saying `egg-info` is a dynamic content and remove it not just from `AVOCADO_OPTIONAL_PLUGINS` but from everywhere in Avocado sources? Anyway this does what is in the commit message, so let me just know whether you would see the `clean-all-eggs` as a suitable approach and if you thing the cons are to great I can merge this instead.